model/models.py

```diff
@@ -102,7 +102,7 @@ def __init__(self, edge_index, edge_type, params=None):
 		self.fc			= torch.nn.Linear(self.flat_sz, self.p.embed_dim)
 
 	def concat(self, e1_embed, rel_embed):
-		e1_embed	= e1_embed. view(-1, 1, self.p.embed_dim)
+		e1_embed	= e1_embed.view(-1, 1, self.p.embed_dim)
```

Comment:
As Max said before please do not reformat the already existing code. This makes it much harder to review your changes

Reply:
fixed, but i looks weird to write it like that with a space between